.github/workflows/ci.yml

```diff
@@ -26,14 +26,19 @@ jobs:
         run: cargo clippy --benches --all-features
       - name: Run clippy (examples)
         run: cargo clippy --examples --all-features
-
   test:
     runs-on: ubuntu-latest
+    services:
+      memcached:
+        image: memcached:latest
+        ports:
+          - 11211:11211
     steps:
       - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
-      - name: Run tests
+      - name: Run parser and connectiontests
         run: cargo test --all-features
-
+      - name: Run integration tests
+        run: cargo test -- --test-threads=1 --ignored
```

Comment:
If we set `--test-threads=1`, do we still need the `serial_test` crate?

Reply:
That would also allow us to remove `serial_test`, but I think it might be better to leave it out of convenience for people who are not familiar with the crate.  It's one thing to know that you have ignored tests that need to be run specifically, and that's made obvious through annotation, but another to know that you have one rogue test that can impact all the others.  I don't really think it adds anything of consequence to the compile time.

It can be moved to a dev dependency though, since this is running tests in CI in a single thread anyway.